editor/components/post-saved-state/style.scss

```diff
@@ -10,17 +10,7 @@
 	}
 
 	.wp-core-ui &.button-link {
-		text-decoration: underline;
-		color: $blue-wordpress;
-		margin-right: $item-spacing;	// needs specificity to override core CSS bleed
-
-		&:focus {
-			outline: none;
```

Comment:
Noting that this style does not exist on the base `.button-link`, but also that it's questionable we'd want it.